docs/common-OBDUtils.js

```
@@ -0,0 +1,765 @@
+
+            return null;
+        }
+      } catch (error) {
+        logOBDUtils('error', 'Error getting protocol:', error);
+
+        return null;
+      }
+    },
+
+    async setProtocol(protocol) {
+      if (!protocolConfig.hasOwnProperty(protocol)) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)